README.md

```diff
@@ -117,17 +122,17 @@ We will set up a deploy key in the Git server that will allow an SSH key we make
  - References:
```

Comment:
I'm guessing that we're going to support them making the keys if they have issues. If not I can add a disclaimer. (Same thing with GitLab instructions, actually.) Up to you.